scaletest/terraform/gcp_cluster.tf

```diff
@@ -26,7 +26,7 @@ resource "google_container_cluster" "primary" {
   monitoring_config {
     enable_components = ["SYSTEM_COMPONENTS"]
     managed_prometheus {
-      enabled = true
+      enabled = false
     }
```

Comment:
nit: Is it enabled or disabled by default?

Reply:
I believe it's disabled by default, but I'd prefer being explicit here.

Reply:
Fair point 👍